opentreemap/importer/models/trees.py

```diff
@@ -91,8 +91,6 @@ class TreeImportRow(GenericImportRow):
         'geom': fields.trees.POINT,
         'width': fields.trees.PLOT_WIDTH,
         'length': fields.trees.PLOT_LENGTH,
-        # TODO: READONLY restore when implemented
```

Comment:
In order to avoid having to pass tests that are unrelated to production, I've been removing tests that are unrelated to production.

The "readonly" tests already were `@skip`ped, but in the interest of simplifying the problem, I removed them. Then, later, I found these comments, which don't make sense in the absence of the tests.